src/pages/workspace/WorkspaceSidebar.js

```diff
@@ -120,7 +120,7 @@ const WorkspaceSidebar = ({
                                 <Tooltip absolute text={translate('workspace.common.edit')}>
                                     <View style={[styles.smallEditIcon, styles.smallAvatarEditIcon]}>
                                         <Icon
-                                            src={Pencil}
+                                            src={Camera}
```

Comment:
Shouldn't this be the `Pencil` icon and the icon in `AvatarWithImagePicker` be the `Camera` icon? This is what I'm seeing:
![Screen Shot 2021-10-01 at 12 00 00 PM](https://user-images.githubusercontent.com/12268372/135585894-611cfa3d-384c-49db-9d3f-a64e007d2ac5.png)

Reply:
I agree with you, those look like they got mixed up.

Reply:
Wow, my bad. I switched the icons a couple of times and didn't notice this 😬 

Thanks for spotting.

Reply:
Oh I see... I made the change locally, took the screenshots and then forgot to make the final commit.